app/src/main/java/com/vero/tasky/agenda/presentation/agenda/AgendaState.kt

```diff
@@ -0,0 +1,15 @@
+package com.vero.tasky.agenda.presentation.agenda
+
+import android.os.Parcelable
+import com.vero.tasky.agenda.domain.model.AgendaItem
+import kotlinx.parcelize.Parcelize
+import java.time.LocalDateTime
+
+@Parcelize
+data class AgendaState(
+    val userShortName: String = "",
+    val month: String = "",
```

Comment:
Here you could have the month, rather than the String (There's definitely something like LocalDate.Month or similar)